src/app/showcase/components/input/showcase-input.component.html

```diff
@@ -36,7 +36,7 @@
     <div class="row mt-1">
         <label class="col-md-3 col-form-label">NumPad</label>
         <div class="col-md-9">
-            <systelab-numpad></systelab-numpad>
+            <systelab-numpad [(numpadValue)]="value" (numpadValueEnter)="doEnter()"></systelab-numpad>
```

Comment:
Use the more standard [(value)] and (change)

Reply:
Done